inventory_level.go

```diff
@@ -0,0 +1,95 @@
+package goshopify
+
+import (
+	"fmt"
+	"time"
+)
+
+const inventoryLevelsBasePath = "inventory_levels"
+
+// InventoryLevelService is an interface for interacting with the
+// inventory items endpoints of the Shopify API
+// See https://help.shopify.com/en/api/reference/inventory/inventorylevel
+type InventoryLevelService interface {
+	List(interface{}) ([]InventoryLevel, error)
+	Adjust(interface{}) (*InventoryLevel, error)
+	Delete(int64, int64) error
+	Connect(InventoryLevel) (*InventoryLevel, error)
+	Set(InventoryLevel) (*InventoryLevel, error)
+}
+
+// InventoryLevelServiceOp is the default implementation of the InventoryLevelService interface
+type InventoryLevelServiceOp struct {
+	client *Client
+}
+
+// InventoryLevel represents a Shopify inventory level
+type InventoryLevel struct {
+	InventoryItemID   int64      `json:"inventory_item_id,omitempty"`
+	LocationID        int64      `json:"location_id,omitempty"`
+	Available         int        `json:"available,omitempty"`
```

Comment:
You should remove `omitempty`, in order to set the stock to 0, otherwise the json marshaller will omit the value.

Reply:
Sorry, about which line you talk? Can not find :(

Reply:
If you create a `InventoryLevel` with `Available` set to 0, you will generate a json without the `available` property that will result to a 400 error.
You just need to remove `omitempty`